### ISSUE TITLE
Loss drops too fast to 0.0

### ISSUE THREAD

**Issue by SungAnGweon** [bermanmaxim/AOWS#1]
I trained train_nas.py with default configuration other than latency output, and keep failed. I checked losses and they soon drop to 0.0 within 2 epochs.
I tried to commented loss in code
```
 for kind in ('min', 'rand'):
            conf = None if kind == 'rand' else minconf
            outp = model(input, configuration=conf)

            loss = misc.soft_cross_entropy(outp['x'], compute_results['max']['prob'].detach())
            compute_results[kind]['soft_loss_numpy'] = loss.detach().cpu().numpy()
            with torch.no_grad():
                hard_loss_numpy = criterion(outp['x'], target).detach().cpu().numpy()
                compute_results[kind]['loss_numpy'] = hard_loss_numpy

            compute_results[kind]['x'] = outp['x'].detach()
            if kind == 'rand':
                compute_results['rand']['decision'] = outp['decision'].cpu().numpy()
            # loss.mean().backward()

```
and print loss of `compute_results['max']['loss_numpy']`, it still drops so fast.

Is it way it is, or is it an abnormal behavior?

Thank you.

**Comment by SungAnGweon** [comment]
Solved. Target was all zero hence it failed training.
Putting shuffle=True in Dataloader fixes the problem.
